lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -30,7 +30,6 @@
 from openedx.core.djangoapps.user_api.tests.factories import UserCourseTagFactory
 
 
-@override_settings(MODULESTORE=TEST_DATA_MOCK_MODULESTORE)
```

Comment:
A note-- there are now a lot of unnecessary imports of TEST_DATA_MOCK_MODULESTORE in the test files.